CLEANUP_IMPLEMENTATION_PLAN.md

```
@@ -0,0 +1,232 @@
+
+## Phase 1: Remove Event Sourcing System (Immediate)
+
+### Files to Delete:
+```bash
+rm lib/wanderer_notifier/event_sourcing/event.ex
+rm lib/wanderer_notifier/event_sourcing/pipeline.ex
+rm lib/wanderer_notifier/event_sourcing/handlers.ex
+rmdir lib/wanderer_notifier/event_sourcing
+```
+
+### Verification:
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)